layouts/search/single.html

```diff
@@ -1,5 +1,16 @@
 {{ define "main" }}
-<section class="search no-sidebar">
-  {{ partial "coveo-atomic.html" .}}
-</section>
+  {{ if .Site.Params.enable_coveo }} 
+    <section class="search no-sidebar">
+      {{ partial "coveo-atomic.html" .}}
+    </section>
+  {{ else }}
+    <div class="content" data-testid="content">
+      <div class="not-found-container" data-testid="not-found-container">
+        <h1 class="info-header">
+          Search functionality is disabled.
```

Comment:
Let's go with this, or just 404. If we want to publish standalone content without search, it makes more sense to 404. For the main content sets we should make it appear that it's temporarily unavailable.

```suggestion
          Search functionality is unavailable.
```